### ISSUE TITLE
Doesn't work on browsers with Gostery extension

### ISSUE THREAD

**Issue by davidbludlow** [phanan/vue-google-signin-button#30]
Gostery ( https://www.ghostery.com/ ) is the world's leading anti-tracking tool (at least, according to [this](https://cliqz.com/en/about)). It has browser extensions for all the major browsers plus many less major ones. 

Unless Gostery is uninstalled or disabled this google sign in button will never fire its `success` or `error` events, making it completely ineffective. 

Gostery has an option to "Trust this site" but that doesn't help at all.

If you want an example, install Gostery on pretty much any browser, then try to login to https://activities.graspablemath.com/login . It will show the google login page, then it will do nothing, leaving you logged out.

**Comment by davidbludlow** [comment]
I think may have found some clues as to the problem by looking at the behavior of edpuzzle.com and socrative.com when Ghostery is active. I don't think either of them use vue-google-signin-button, but I think the clues are still valuable. See https://www.screencast.com/t/ei7REiUPVBM .

**Comment by phanan** [comment]
What this plugin does is actually only render the sign-in button, no more no less. It seems to me the problem lies on Ghostery side (I've never used it before btw).

**Comment by davidbludlow** [comment]
I agree that it is Ghostery's fault, but this still makes the websites that use vue-google-signin-button appear at fault to the user. I am a web developer and it still took me 6 hours to figure out that telling Gostery to "Trust this site" wasn't enough, I needed to disable the plugin. 

I propose that we gang up on Ghostery and try to convince them to change their code, or failing that, we figure out a way to detect when Ghostery is causing problems and warn the user to disable the plugin.
